tap2junit/__main__.py

```diff
@@ -2,7 +2,7 @@
 import os
 import platform
 from junit_xml import TestSuite, TestCase
-from tap13 import TAP13 as tap13
+from .tap13 import TAP13 as tap13
```

Comment:
just to check, this work on py2 and py3?

Reply:
Thanks,  Changed to absolute import.